booster-app/src/main/java/de/neuefische/boosterapp/controller/BoosterController.java

```diff
@@ -26,10 +25,10 @@ public BoosterController(BoosterService boosterService) {
     }
 
 
-//    @PutMapping("{ownerId}/random")
-//    public Booster getRandomBoosterOfType(@PathVariable String ownerId, @RequestBody BoosterType randomBoost) {
-//        return boosterService.getRandomBooster(randomBoost, ownerId);
-//    }
+    @PutMapping("list")
+    public Booster getRandomBooster(@RequestParam  String owner, @RequestBody BoosterType randomBoost) {
```

Comment:
get random booster with put mapping is very strange. would be better to use a get and put BoosterType into the request params

Reply:
with list i would expect to get a list. I think random would be better